src/plays/password-generator/Readme.md

```diff
@@ -0,0 +1 @@
+# Password Generator
```

Comment:
Put some explanation in this. See other play readmes as reference.

Reply:
yes the explanations are added.